api/GlobalEventHandlers.json

```
@@ -170,25 +170,37 @@
                 "version_added": "79"
               },
               {
-                "version_added": true,
+                "version_added": "81",
                 "alternative_name": "onwebkitanimationend"
+              },
+              {
+                "version_added": "43",
+                "version_removed": "79",
+                "partial_implementation": true,
+                "notes": "Before Chrome 79, this event handler was only supported on the <code>Window</code> interface."
```

The versions are already given by `version_added` and `version_removed`, so:

```suggestion
                "notes": "Only supported on the <code>Window</code> interface."
```